WordPress/gutenberg
# Image block: Add aspect ratio support to lightbox

## What?

This PR makes it so the lightbox zoom animation functions as expected when configuring a custom aspect ratio on an image.

## Why?

Addresses #52192
We want the lightbox to be compatible with all of the image block's features.

## How?

It calculates the aspect ratio of the resized image and uses it to calculate the target zoom dimensions.

## Testing Instructions




1. Ensure interactivity API experiments are enabled
2. Add an image to a post
3. Customize the image's `aspect ratio` in block settings
4. Activate the lightbox under Advanced > Behaviors > Lightbox. Make sure the `zoom` animation is selected.
5. Publish, view the post, and click on the image — it should zoom in as expected.

*If possible, please also test on a mobile device.*

### Testing Instructions for Keyboard

N/A

## Screenshots or screencast 

### Before

https://github.com/WordPress/gutenberg/assets/5360536/08c395b4-afbb-4a16-a518-034d8935ea8c

### After

https://github.com/WordPress/gutenberg/assets/5360536/c66020b0-8e75-4187-aefe-8007c2ac5dcb
